zebrad/src/components/mempool.rs

```diff
@@ -224,7 +234,9 @@ impl Service<Request> for Mempool {
                     if let Ok(tx) = r {
                         // Storage handles conflicting transactions or a full mempool internally,
                         // so just ignore the storage result here
-                        let _ = storage.insert(tx);
+                        let _ = storage.insert(tx.clone());
```

Comment:
Correctness: Do we want to gossip transactions that are rejected by `storage` or `remove_expired_transactions`?